jest/jest.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = async () => ({
+	rootDir: "..",
+	moduleFileExtensions: ["tsx", "ts", "js"],
+	testEnvironment: "jsdom",
+	testMatch: ["<rootDir>/src/**/__tests__/**/*.spec.[jt]s?(x)"],
+	maxConcurrency: 10,
+	collectCoverageFrom: [
+		"<rootDir>/src/**/*.{js,jsx,ts,tsx}",
+		// Generic exclusions
+		"!**/__tests__/**/*",
+		"!**/stories/**/*",
+		"!**/*{M,m}ock{*,*/**/*}",
+		"!**/*{E,e}num{*,*/**/*}",
```

Comment:
Do we need all these?

Reply:
they are not needed indeed, have removed them